packages/invoices/src/InboundShipment/DetailView/reducer/reducer.ts

```diff
@@ -155,20 +155,18 @@ export const reducer = (
         }
 
         case DocumentActionType.Merge: {
-          const { draft } = state;
-
-          Object.keys(draft).forEach(key => {
-            // TODO: Sometimes we want to keep the user entered values?
-            if (key === 'items' || key === 'lines') return;
-            draft[key] = data[key];
-          });
+          state.draft = {
+            ...state.draft,
+            ...data,
+            status: state.draft.status,
+          };
```

Comment:
This is equivalent (pretty sure 😁 )

Taking the `data` fields over `draft`, and then later on, `items` gets assigned correctly.

The `status` was just because of type annoyances..